gems/graphql/1.12/graphql.rbs

```diff
@@ -191,6 +191,7 @@ module GraphQL
   class Error < StandardError
   end
   class ExecutionError < Error
+    def initialize: (String message, ?ast_node: untyped, ?options: Hash[untyped, untyped]?, ?extensions: Hash[untyped, untyped]?) -> void
```

Comment:
The `options` keyword seems deprecated.

RBS has both recommended and documented implications.
I don't think deprecated arguments should be written in the signature.

https://github.com/rmosolgo/graphql-ruby/blob/4ecf14faf77262fcfe4ff8231f131e6b7dadc09b/lib/graphql/execution_error.rb#L15

Reply:
@ksss Thank you for the review! Removed the deprecated `options` parameter in ac6de66cb27e99b4a93bb05693bc17123e1fdd09